src/pages/Tracking/Tracking.jsx

```diff
@@ -67,6 +67,13 @@ class Tracking extends Component {
 				visible: false,
 				colour: 'red',
 				tracks: pointsMap[tracker.id],
+				max_error_radius: tracker.max_error_radius,
+				location_method: tracker.location_method,
+				loading: {
+					status: false,
+					name: false,
+					colour: false,
```

Comment:
why are these in `loading`?. `loading` strikes me as a boolean variable if the deer data has come back or not. More intuitive name might be helpful, or I might be missing something?